apps/system/js/core.js

```diff
@@ -124,7 +124,8 @@
             'js/devtools/remote_debugger.js',
             'js/devtools/developer_hud.js',
             'js/devtools/devtools_auth.js',
-            'js/devtools/devtools_auth_dialog.js'
+            'js/devtools/devtools_auth_dialog.js',
+            'js/sync/sync_iac_api.js'
```

Comment:
Why add Sync IAC API in this PR? I would expect you add an IAC API just for Places?